README.md

```diff
@@ -1,32 +1,39 @@
-# Rector - Upgrade Your Legacy App to a Modern Codebase
+# Rector
 
-Rector is a **rec**onstruc**tor** tool - it does **instant upgrades** and **instant refactoring** of your code.
-Why refactor manually if Rector can handle 80% for you?
+## What is it?
+Rector is a command line tool that can upgrade and refactor your PHP code from legacy code to modern, clean code **instantly**. 
+<br>
+
+## How can rector help you?
+- You want to upgrade from PHP 5.3 to 7.4?  Well, Rector can do that for you.  
+- You want to switch frameworks? Well, Rector can do that for you as well. 
+<br>
 
+## Want to see?
 [![Build Status](https://img.shields.io/travis/rectorphp/rector/master.svg?style=flat-square)](https://travis-ci.org/rectorphp/rector)
 [![Coverage Status](https://img.shields.io/coveralls/rectorphp/rector/master.svg?style=flat-square)](https://coveralls.io/github/rectorphp/rector?branch=master)
 [![Downloads](https://img.shields.io/packagist/dt/rector/rector.svg?style=flat-square)](https://packagist.org/packages/rector/rector)
-
-
 ![Rector-showcase](docs/images/rector-showcase-var.gif)
+<br>
 
+## What can it do technical?
+- Rename classes, methods, properties, namespaces or constants
+- Complete parameter, var or return type declarations based on static analysis of your code
+- Complete PHP 7.4 property type declarations
+- Refactor Laravel facades to dependency injection
 <br>
 
 ## Sponsors
-
 Rector grows faster with your help, the more you help the more work it saves you.
 Check out [Rector's Patreon](https://www.patreon.com/rectorphp). One-time donation is welcomed [through PayPal](https://www.paypal.me/rectorphp).
-
+<br>
 Thank you:
-
+<br>
 <a href="https://spaceflow.io/en"><img src="/docs/images/spaceflow.png"></a>
-
 <br>
 
 ## Open-Source First
-
-Rector **instantly upgrades and instantly refactors the PHP code of your application**. It supports all modern versions of PHP and many open-source projects:
-
+Rector supports all modern versions of PHP and many open-source projects:
```

Comment:
:+1: 

Yes, "instant upgrade" and "instant migration" keywords should be kept. That's new word that describes what is it :)

Reply:
👍  Yeah, that's better than what I wrote.